imod/mf6/simulation.py

```diff
@@ -1046,15 +1056,23 @@ def __repr__(self) -> str:
             content = attrs + ["){}"]
         return "\n".join(content)
 
-    def _generate_gwfgwt_exchanges(self):
+    def _generate_gwfgwt_exchanges(self) -> list[GWFGWT]:
         flow_models = self.get_models_of_type("gwf6")
```

Comment:
Please add some docstring to this function:

I was first confused that multiple transport models could be coupled to a flow model, but this is of course what happens for multiple species.